app/cafe.py

```diff
@@ -0,0 +1,30 @@
+import datetime
+from app.errors import (NotVaccinatedError,
+                        OutdatedVaccineError, NotWearingMaskError)
+
+
+class Cafe:
+    def __init__(self, name: str) -> None:
+        self.name = name
+
+    def visit_cafe(self, visitor: dict) -> str:
+        # Check if visitor is vaccinated
+        if "vaccine" not in visitor:
+            raise NotVaccinatedError(
+                f"{visitor.get("name", "Visitor")}"
```

Comment:
There is a mistake in using quotes inside the f-string. Since the outer quotes are double quotes, the inner quotes should be single quotes to avoid syntax errors.